internal/checker/checker.go

```diff
@@ -10826,39 +10967,15 @@ func (c *Checker) isPropertyAccessible(node *ast.Node, isSuper bool, isWrite boo
 }
 
 func (c *Checker) containerSeemsToBeEmptyDomElement(containingType *Type) bool {
-	if c.compilerOptions.Lib == nil || slices.Contains(c.compilerOptions.Lib, "lib.dom.d.ts") {
+	return !slices.Contains(c.compilerOptions.Lib, "dom") && everyContainedType(containingType, hasCommonDomTypeName) && c.isEmptyObjectType(containingType)
```

Comment:
The `dom` thing here is a bugfix; `lib` can never contain `dom`, only `lib.dom.d.ts`.

Reply:
Will fix.